### PR TITLE
Clear the decks: Part 1

### PR DESCRIPTION
**Pull request by keirog**
- Move port variable into config file
- Send errors to Sentry
- Remove redundant timeout assignment
- Improve o-table sorting
	- Makes use of the [o-table data types](https://github.com/financial-times/o-table#sorting). Also enables custom sorting for a future commit.
- Log when we get data from SparkPost
- Show the client name to match the route
	- (If you’re on /analytics/eme, Show “eme” in the title)
- Rename function to be clearer

Sorry for the big PR; might be easier to review per commit. Normally I’d prefer smaller PRs but there’s lots to get through from the Keen-rush 😬✌️

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "@financial-times/health-check": "^1.4.1",
     "@financial-times/n-internal-tool": "^2.3.1",
     "@financial-times/n-logger": "^6.1.0",
+    "@financial-times/n-raven": "^3.0.4",
```
Comment (j-mes) [inline package.json]: What does `n-raven` do?

File: server/views/dashboard.html [hunk 2]
```diff
@@ -37,13 +41,13 @@ <h1 class="o-typography-product-heading-level-1">
...
-							{{else~}}
-								{{name}}
-							{{/if~}}
-							</th>
+								<th scope="col" role="columnheader"{{#if type}} data-o-table-data-type="{{type}}"{{/if}}>
+								{{~#if abbreviation~}}
+									<abbr title="{{name}}">{{abbreviation}}</abbr>
+								{{else~}}
+									{{name}}
+								{{/if~}}
```
Comment (j-mes) [inline server/views/dashboard.html]: What does the `~` do?

File: config.js [hunk 3]
```diff
@@ -6,5 +6,6 @@ module.exports = {
 	hoursAgo: Number(process.env.HOURS_AGO || 24),
 	host: process.env.HOST || 'localhost',
 	pollingFrequency: Number(process.env.POLLING_FREQUENCY || 30000),
+	port: Number(process.env.PORT || 3000),
```
Comment (j-mes) [inline config.js]: Oooh, this looks good

File: package.json (hunk 1 above)
Comment (keirog) [reply package.json]: Good question. It’s a client for communicating with Sentry (although it does need updating as they don’t call it raven anymore!)

File: server/views/dashboard.html (hunk 2 above)
Comment (keirog) [reply server/views/dashboard.html]: It‘s a very handy way to [strip whitespace from handlebars templates](https://handlebarsjs.com/expressions.html#whitespace-control)

Comment (j-mes) [reply server/views/dashboard.html]: That is neat!

**Pull request merged**